### PR TITLE
Audit log WIP

### PR DESCRIPTION
**Pull request by rjkip**

### CODE REVIEW

**Comment by rjkip** [comment]
:eyeglasses: @DRvanR

File: src/Surfnet/StepupMiddlewareClient/Identity/Dto/AuditLogSearchQuery.php [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            array_filter(
+                [
+                    'institution'    => $this->institution,
+                    'identityId'           => $this->identityId,
+                    'orderBy'        => $this->orderBy,
+                    'orderDirection' => $this->orderDirection,
+                    'p'              => $this->pageNumber,
+                ],
+                function ($value) {
+                    return !is_null($value);
```
Comment (DRvanR) [inline src/Surfnet/StepupMiddlewareClient/Identity/Dto/AuditLogSearchQuery.php]: Question: Why the explicit check for `null`?

**Comment by DRvanR** [comment]
:+1:

**Comment by rjkip** [comment]
Better?

**Comment by DRvanR** [comment]
much bettah... :+1:

**Pull request merged**